BUILD.md

```diff
@@ -22,6 +22,21 @@ go build ./...
 go test ./...
 ```
 
+# Regenerating Generated Code
+
+To generate the deepcopy functions, clientset, listers and informers, run the following command:
+
+```
+GOPATH=<path to go base> ./scripts/update-codegen.sh
+```
+
+To regenerate the deepcopy code need for our CRD code, run the following commands:
+
+```
+go install ./cmd/deepcopy-gen/
+$GOPATH/bin/deepcopy-gen --input-dirs ./netmesh/model/netmesh --go-header-file conf/boilerplate.txt --bounding-dirs ./netmesh/model/netmesh -O zz_generated.deepcopy -o $GOPATH/src
```

Comment:
This separate run of `deepcopy-gen` is required because our CRDs make use of the generated go structures from the protobuf files in `netmesh/model/netmesh`.